config/orion.yaml

```diff
@@ -48,6 +48,25 @@ aggregators:
     hosts : "agg-31"
     group : l3-agg
 
+samplers:
+# This is the configuration for the LDMS sampler daemons
+  - names       : *sampler-hosts
+    hosts       : *sampler-hosts
+    ports       : *sampler-ports
+    group       : samplers
+    config :
+      - name        : meminfo
+        interval    : "1.0s:0ms" # Format is interval:offset
+        job_count   : 4
```

Comment:
What's the 'job_count?'

Reply:
I believe it's for stream sampling, so it's not relevant to the meminfo sampler. I had included job_count and stream originally because I wanted to test null variables and integer handling, but these both can be removed from the config example if you'd like.

Reply:
yes, because they're not used by those particular sampler instances. The job_count only applies to the slurm sampler. If you want to add the slurm sampler to the examples, that would be fine.